int8 object_type # Type of object detected (0: unknown, 1: blue, 2: orange, 3: green, 4: red)
uint32 x_coordinate   # X coordinate of the object in the image
uint32 y_coordinate   # Y coordinate of the object in the image